packages/frontend/src/components/composer/ComposerMessageInput.tsx

```diff
@@ -46,10 +46,9 @@ export default class ComposerMessageInput extends React.Component<
     this.insertStringAtCursorPosition =
       this.insertStringAtCursorPosition.bind(this)
 
-    this.saveDraft = debounce(() => {
-      const { text, chatId } = this.state
+    this.saveDraft = throttle((text: string, chatId: number) => {
```

Comment:
Am I understanding this correctly that the switching from `const { text, chatId } = this.state` to `(text: string, chatId: number) =>` has no functional significance?